jme3-core/src/main/java/com/jme3/audio/AudioBuffer.java

```diff
@@ -87,8 +87,12 @@ public String toString(){
     /**
      * Update the data in the buffer with new data.
      * @param data
+     * @throws IllegalArgumentException if the provided buffer is not a direct buffer
      */
     public void updateData(ByteBuffer data){
+        if (!data.isDirect())
```

Comment:
My personal opinion/style would want `{}s` even for one-line statements, but I don't know:
- What our jme desired standard would be and
- how our current code looks right now

Reply:
Yeah, especially in this case where the line is pretty long.

Reply:
Yeah, `{}` is preferred.

Reply:
I got impatient and standardized the coding style.